Cargo.toml

```
@@ -1,18 +1,19 @@
 [package]
-name = "fuser"
-description = "Filesystem in Userspace (FUSE) for Rust"
+name = "mountpoint-s3-fuser"
+description = "A fork of filesystem in Userspace (FUSE) implementation for Rust for use in Mountpoint S3 only"
```

Maybe something like this:
```suggestion
description = "A fork of fuser - Filesystem in Userspace (FUSE) for Rust - only for use in Mountpoint for Amazon S3"
```

---

This still need updating